frontend/src/index.ts

```diff
@@ -89,7 +89,7 @@ export class App extends LiteElement {
       AppStateService.updateOrgSlug(this.viewState.params.slug || null);
     }
     if (authState) {
-      this.authService.saveLogin(authState);
+      this.authService.saveLogin(authState as unknown as Auth);
```

Comment:
Don't think the coercion is necessary
```suggestion
      this.authService.saveLogin(authState);
```

Reply:
Hmm, odd, I was getting a Typescript build error without the coercion, but it no longer errors...